src/lib/components/molecules/column-chart/columnchart.stories.jsx

```diff
@@ -0,0 +1,50 @@
+import { ColumnChart } from '.'
+
+const meta = {
+  title: 'Molecules/ColumnChart',
+  component: ColumnChart,
+}
+
+export default meta
+
+export const Default = {
+  args: {
+    domain: [300, -100],
+    chartHeight: 400,
+    columnWidth: 20,
+    columns: [
+      {
+        value : 100,
+        id: "con",
+        color: "blue"
+      },
+      {
+        value : -100,
+        id: "lab",
+        color: "red"
+      },
+    ]
+  },
+  render: (args) => <ColumnChart {...args} />,
+}
+
+export const Councillors = {
+  args: {
+    domain: [300, -100],
```

Comment:
For user ease - could I suggest that domain is an object with named properties `min` and `max`. I can see people getting these easily mixed up

Reply:
Good call. Done.